mobile/package.json

```diff
@@ -43,6 +43,8 @@
     "react-native": "0.63.2",
     "react-native-animatable": "^1.3.3",
     "react-native-appearance": "^0.3.4",
+    "react-native-google-fit": "^0.16.2",
+    "react-native-health": "^1.8.0",
```

Comment:
didn't we already have a health API library installed?

Reply:
It looks like this might be the root of the issue, we have two libraries installed that create AppleHealthKit objects in the compiled code for iOS, which is at least one of the build errors I see. Is there a reason we're using the new package over the existing one?

Reply:
Oh right, we use this package for getting access to the health permission, the old package we use is only available for few access instead of the all we need, and the old package might generate some error when calling the API

Reply:
I see. So we can just use the new one?